scripts/import_cldr.py

```diff
@@ -739,14 +740,28 @@ def parse_calendar_datetime_skeletons(data, calendar):
 
 def parse_number_symbols(data, tree):
     number_symbols = data.setdefault('number_symbols', {})
-    for symbol_elem in tree.findall('.//numbers/symbols'):
-        if _should_skip_number_elem(data, symbol_elem):  # TODO: Support other number systems
+    for symbol_system_elem in tree.findall('.//numbers/symbols'):
+        number_system = symbol_system_elem.get('numberSystem')
+        if not number_system:
             continue
 
-        for elem in symbol_elem.findall('./*'):
-            if _should_skip_elem(elem):
+        for symbol_element in symbol_system_elem.findall('./*'):
+            if _should_skip_elem(symbol_element):
                 continue
-            number_symbols[elem.tag] = str(elem.text)
+
+            number_symbols.setdefault(number_system, {})[symbol_element.tag] = str(symbol_element.text)
+
+
+def parse_numbering_systems(data, tree):
+    numbering_systems = data.setdefault('numbering_systems', {})
+    default_number_system_node = tree.find('.//numbers/defaultNumberingSystem')
+
+    if default_number_system_node is not None:
+        numbering_systems['default'] = str(default_number_system_node.text)
```

Comment:
I think it might be better to just have `default_numbering_system` in the `data` by itself, because otherwise `numbering_systems`'s type will be a silly `dict[str, str | list[str]]`.

Reply:
I think it might be worth to have a separate `default_numbering_system` since I foresee that to be a pretty hot path, and saving one dict lookup might compound :)